packages/project-model/package.json

```diff
@@ -32,6 +32,17 @@
     "@types/node": "^12.12.0",
     "@types/vscode": "^1.46.0"
   },
+  "eslintConfig": {
```

Comment:
I added a few eslint rule overrides.
* import/order - this is just for convenience, we can remove it eventually
* no-this-alias - this is necessary. some patterns (like using generator functions) require the `const self=this` alias
* no-non-null-assertion: this is necessary. there is a reason this assertion exists. especially when type-guards cannot be used (or require convoluted rewrites).
* no-case-declarations: supporting this rule would require rewriting most logic of the interactive_cli. I'm not opposed to doing that, but it doesn't add any value
* prefer-const: we could activate this rule eventually. there are some parts of the code that use "let" to allow quickly changing values when testing (by adding an extra assignment)
* no-unused-expressions: This gets in the way when using Wallaby.js